src/main/integrator.xml

```diff
@@ -7,6 +7,8 @@ Copyright 2006 IBM Corporation
 See the accompanying LICENSE file for applicable license.
 -->
 <project name="dita.integrator" default="integrate">
+
+  <echo level="warn">WARN: Ant integrator has been deprecated. Use dita command instead.</echo>
```

Comment:
Since the `dita uninstall` command requires a plug-in ID argument, should we even mention it here as an alternative to the Ant integrator? 🤔 

If you delete plug-ins & run `dita install`, it uninstalls those, so maybe something like this would be sufficient?

```suggestion
  <echo level="warn">WARN: The Ant integration process is deprecated. Use the `dita install` command instead.</echo>
```